_layouts/styleguide.html

```diff
@@ -20,6 +20,9 @@
         {% endunless %}
       {% endif %}
       <h1 id="{{ page.title | slugify }}">{{ page.title }}</h1>
+      <span class="components-library-link">
+        <a href="{{ page.component_url }}">View in Component Library</a>
```

Comment:
So based on my suggestion in https://github.com/18F/web-design-standards-docs/pull/366#issuecomment-317887508, you could change this line to:

```html
<a href="https://components.standards.usa.gov/components/detail/{{ page.fractal_component }}">View in Component Library</a>
```

That way individual components just have to define e.g. `fractal_component: buttons` in their front matter instead of a big 'ol URL.

Oh! I just noticed you've also defined a `components_url` in your `_config.yml`, so maybe the above HTML could even be something like:

```html
<a href="{{ site.components_url }}/components/detail/{{ page.fractal_component }}">View in Component Library</a>
```

That way if we eventually decide to have e.g. a staging site for the fractal components + docs, we can have them link to each other, rather than the docs always hard-linking themselves to the production fractal site.